### PR TITLE
Support direct reference of includes types in mapping sourcepaths

### PR DESCRIPTION
**Pull request by cmoesel**
Due to legacy reasons for how includes types are represented in paths in SHR FHIR export, the way you reference them in the mapping file is *different* than how you might reference them in a data elements file.

In short, you replace the original identifier with the new includes type identifier.  For example, if in your definition you had:

```
Components.ObservationComponent
  includes 0..1 SystolicPressure
  includes 0..1 DiastolicPressure
```

You would reference those in mappings as:

```
Components.SystolicPressure maps to ...
Components.DiastolicPressure maps to ...
```

This works with a corresponding branch / PR on `shr-fhir-export` and the `bp_component_mapping` branch of `shr_spec`.

### CODE REVIEW

**Comment by cmoesel** [comment]
Please do not merge.  Only approve (if appropriate).

**Review by kjmahalingam — COMMENTED** [review]
See related comment here: https://github.com/standardhealth/shr-fhir-export/pull/151#pullrequestreview-268337338

**Review by kjmahalingam — APPROVED** [review]

**Pull request merged**
